src/features/expenses/ui/CreateExpense/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { Field, Form, Formik } from 'formik';
+import { type FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useStore } from '@shared/lib/useStore';
+import { Button } from '@shared/ui/Button';
+import { Select } from '@shared/ui/Select';
+import { AmountAndCurrency, Datepicker } from '@entities/expenses';
+import { Icon } from '@shared/icons/Icon';
+import { useExpensesMutation } from '@features/expenses/model/useExpensesMutation';
+
+const initialValues = {
+  category: '',
+  currency: '',
+  amount: '',
+  month: '',
+  day: '',
+  year: '',
+  details: '',
+};
+
+export const CreateExpense: FC = () => {
+  const { categoriesStore } = useStore();
+
+  const categoriesOptions = categoriesStore.getCategories().map((category) => {
+    return category.name;
+  });
+
+  const { mutation, onFormSubmit } = useExpensesMutation();
```

Comment:
```suggestion
  const { isPending, onFormSubmit } = useExpensesMutation();
```
In my opinion, it can be enough